.github/workflows/test.yml

```diff
@@ -0,0 +1,106 @@
+name: "Test aws-robomaker-github-actions"
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+      - 'releases/*'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test_robomaker-sample-app-ci:
+    runs-on: ubuntu-latest
+    
+    strategy:      
+      matrix:
+        distro: ['kinetic', 'melodic']
+        gazebo: [7, 9]
+        include:
+        - distro: kinetic
+          gazebo: 7
+          ubuntu_distro: xenial
+        - distro: kinetic
+          gazebo: 9
+          ubuntu_distro: xenial
+        - distro: melodic
+          gazebo: 9
+          ubuntu_distro: bionic
+        exclude:
+        - distro: 'melodic'
+          gazebo: 7      
+      
+    container: 
+      image: rostooling/setup-ros-docker:ubuntu-${{ matrix.ubuntu_distro }}-ros-${{ matrix.distro }}-ros-base-latest
+      
+    # NOTES: We run 2 tests for robot_ws and simulation_ws
+    # The test steps are duplicated because github workflow does not support reusable step (with parameter yet)
+    # We also can use matrix to run tests with different parameters but it is overkill for this
+    steps:    
+    - name: Checkout code
+      uses: actions/checkout@v1
+    - name: Setup permissions
+      run: |
+        # Due to user permisson issue, calling chown is necessary for now
+        # Related issue: https://github.com/actions/checkout/issues/47
+        # TODO(ros-tooling/setup-ros-docker#7): 
+        sudo chown -R rosbuild:rosbuild "$HOME" .      
+    - name: Setup nodeJS
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    - name: Install nodeJS packages
+      run: npm install
+      working-directory: ./robomaker-sample-app-ci
+    - name: Build test script
+      run: npm run build
+      working-directory: ./robomaker-sample-app-ci
+      
+    # Trigger to run robomaker-sample-app-ci action script
+    - name: Build and bundle robot_ws      
+      run: node ./robomaker-sample-app-ci/dist/index.js
+      env:
+        INPUT_ROS-DISTRO: ${{ matrix.distro }}
+        INPUT_GAZEBO-VERSION: ${{ matrix.gazebo }}
+        INPUT_WORKSPACE-DIR: ./.github/workflows/data/robot_ws
+        INPUT_GENERATE-SOURCES: true
+    # Expectation: bundle file, build files, and dependencies all exist
+    - name: Check robot_ws file existence
+      id: check_robot_ws_files
+      uses: andstor/file-existence-action@87d74d4
```

Comment:
Can we use a version tag (eg. `v1.0.1`) instead of a commit hash?

Reply:
I'll accept the rational [here](https://github.com/aws-robotics/aws-robomaker-github-actions/pull/9#discussion_r488915550).